lib/test.gi

```diff
@@ -536,7 +543,7 @@ end);
 ## 
 ##    testOptions := rec()   : Options to pass on to Test
 ##    earlyStop := false     : Stop once one test fails
-##    showProgress := true   : Show progress
+##    showProgress := "some" : Show some progress
```

Comment:
How does that ease maintenance? Do I really need to rebase again to get rid of some comments? Also, which of them? Why are those in 541/542 more helpful? Also, somebody put those comments there, so if I remove them now, that person might disagree with the change... Huh